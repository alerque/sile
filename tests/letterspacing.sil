\begin[papersize=a5]{document}

No letterspacing.

\set[parameter=document.letterspaceglue,value=1ex]

Tracking before punctuation?

\set[parameter=document.letterspaceglue,value=-0.2ex]

Negative tracking.

\end{document}
