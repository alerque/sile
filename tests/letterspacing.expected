Set paper size 	419.5275636	595.275597
Begin page
Mx 	40.9764
My 	37.3321
Set font 	Gentium Plus;10;400;;normal;;LTR
T	49 82	(No)
Mx 	55.1691
T	79 72 87 87 72 85 86 83 68 70 76 81 74	(letterspacing)
Mx 	109.1877
T	17	(.)
Mx 	40.9764
My 	49.3321
T	55	(T)
Mx 	51.0252
T	85	(r)
Mx 	59.5262
T	68	(a)
Mx 	68.6570
T	70	(c)
Mx 	77.5682
T	78	(k)
Mx 	86.9871
T	76	(i)
Mx 	94.2381
T	81	(n)
Mx 	104.2967
T	74	(g)
Mx 	111.8320
T	69	(b)
Mx 	121.4512
T	72	(e)
Mx 	130.6113
T	73	(f)
Mx 	138.2627
T	82	(o)
Mx 	147.8330
T	85	(r)
Mx 	156.3340
T	72	(e)
Mx 	163.6105
T	83	(p)
Mx 	173.4494
T	88	(u)
Mx 	183.2883
T	81	(n)
Mx 	193.3469
T	70	(c)
Mx 	202.2580
T	87	(t)
Mx 	210.2414
T	88	(u)
Mx 	220.0803
T	68	(a)
Mx 	229.2111
T	87	(t)
Mx 	237.1945
T	76	(i)
Mx 	244.4455
T	82	(o)
Mx 	254.0158
T	81	(n)
Mx 	259.5334
T	34	(?)
Mx 	40.9764
My 	61.3321
T	49	(N)
Mx 	46.5574
T	72	(e)
Mx 	50.2684
T	74	(g)
Mx 	54.2381
T	68	(a)
Mx 	57.9197
T	87	(t)
Mx 	60.4539
T	76	(i)
Mx 	62.2557
T	89	(v)
Mx 	66.2791
T	72	(e)
Mx 	73.5733
T	87	(t)
Mx 	76.1075
T	85	(r)
Mx 	79.1592
T	68	(a)
Mx 	82.8409
T	70	(c)
Mx 	86.3028
T	78	(k)
Mx 	90.2725
T	76	(i)
Mx 	92.0743
T	81	(n)
Mx 	96.6836
T	74	(g)
Mx 	101.5616
T	17	(.)
Mx 	207.4176
My 	553.7327
T	20	(1)
End page
Finish
